### PR TITLE
fix: [CDS-43892]: fix PDC connector feedback issues

### PR DESCRIPTION
**Pull request by radujomir**
### Summary

Fix PDC connector details feedback issues

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by abeg777** [comment]
retrigger build

**Comment by radujomir** [comment]
retrigger titlecheck

**Comment by radujomir** [comment]
trigger cypress

**Comment by radujomir** [comment]
retrigger jest

**Comment by radujomir** [comment]
retrigger coverage

**Comment by radujomir** [comment]
trigger cypress

File: src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx [hunk 1]
```diff
@@ -1007,7 +1007,7 @@ const getPDCConnectorHosts = (connector: ConnectorInfoDTO) => {
     ? [
         {
           label: 'connectors.pdc.hosts',
-          value: hosts.join(', ')
+          value: hosts.join(',\r\n')
```
Comment (mayankVermaHarness) [inline src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx:1010]: what does the `\r\n` do?

Comment (radujomir) [reply src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx:1010]: A carriage return (\r) makes the cursor jump to the first column (begin of the line) while the newline (\n) jumps to the next line and might also to the beginning of that line. So to be sure to be at the first position within the next line one uses both.

**Comment by radujomir** [comment]
trigger rebase

**Comment by radujomir** [comment]
trigger cypress

**Comment by radujomir** [comment]
retrigger cypress

**Review by mayankVermaHarness — DISMISSED** [review]

**Review by litvinovstas — APPROVED** [review]

**Review by avinashmadhwani02 — APPROVED** [review]

**Pull request merged**